src/main/java/org/broadinstitute/consent/http/resources/InstitutionResource.java

```diff
@@ -0,0 +1,111 @@
+package org.broadinstitute.consent.http.resources;
+
+import com.google.gson.Gson;
+import com.google.inject.Inject;
+import io.dropwizard.auth.Auth;
+import java.util.List;
+import javax.annotation.security.PermitAll;
+import javax.annotation.security.RolesAllowed;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.DELETE;
+import javax.ws.rs.GET;
+import javax.ws.rs.POST;
+import javax.ws.rs.PUT;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.Response;
+import org.broadinstitute.consent.http.models.AuthUser;
+import org.broadinstitute.consent.http.models.Institution;
+import org.broadinstitute.consent.http.models.User;
+import org.broadinstitute.consent.http.service.InstitutionService;
+import org.broadinstitute.consent.http.service.UserService;
+import org.broadinstitute.consent.http.util.InstitutionUtil;
+
+@Path("api/institutions")
+public class InstitutionResource extends Resource {
+  private final UserService userService;
+  private final InstitutionService institutionService;
+  private final InstitutionUtil institutionUtil = new InstitutionUtil();
+
+  @Inject
+  public InstitutionResource(UserService userService, InstitutionService institutionService) {
+    this.userService = userService;
+    this.institutionService = institutionService;
+  }
+
+  @GET
+  @Produces("application/json")
+  @PermitAll
+  public Response getInstitutions(@Auth AuthUser authUser) {
+    try{
+      User user = userService.findUserByEmail(authUser.getName());
+      Boolean isAdmin = institutionUtil.checkIfAdmin(user);
+      Gson gson = institutionUtil.getGsonBuilder(isAdmin);
+      List<Institution> institutions = institutionService.findAllInstitutions();
+      return Response.ok().entity(gson.toJson(institutions)).build();
```

Comment:
TOL: Normally, we'd just return the list as the entity. In this case, we **need** to run it through Gson so it handles the exclusion strategy. I think this could use a comment to explain why we're doing it this way. Same feedback for the other `PermitAll` method(s).